department-of-veterans-affairs/content-build
# VACMS-14071 Footer links fix

## Description
- Ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14071

After pulling in Drupal data to populate the footer, we found out that we weren't creating the relative URLs correctly for the injected header scenario. For instance, if `/find-forms` was coming from the 

## Testing done & Screenshots
Locally, I added a dummy URL into the array of footer links coming back from GraphQL.
<img width="544" alt="Screenshot 2023-07-13 at 2 26 54 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/ecc278f8-23ef-402b-96f8-2524a5071b35">

Then confirmed it was being populated correctly in the footer data coming into vets-website (injected header scenario).
<img width="699" alt="Screenshot 2023-07-13 at 2 27 20 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/d423fb76-e7f3-4fab-b820-426568ecca52">
<img width="538" alt="Screenshot 2023-07-13 at 2 27 27 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/3fb2adb1-ef5a-449e-821d-bcb51340e6e9">

## QA steps
Add a dummy relative link (such as `/find-forms`) into the footer in Drupal. Check the injected header scenario to make sure it works (`https://staging.va.gov/find-forms`).

## Acceptance criteria
- [ ] Injected footer template processes entity-reference (ie relative?) links like the header does, so that va.gov/ links are rendered on the correct base url in injected sites
